core/tests/linter_tests/invalid_alignment_of_tags.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
Missing copyright?

Reply:
I don't think we have copyright for HTML files

Reply:
Okay got it.

Reply:
(Also, add a top comment what is wrong in this file too)